getpathsplus.py

```diff
@@ -57,14 +57,6 @@
 # get python version
```

Comment:
Should the getclicommand definition be included here?

Reply:
And missing the lines that'd call it too?

```
# get cli location from properties, check that cli is there if not ERROR and stop
clicommand=getclicommand()
```

Reply:
only needed if we use the cli in the tool